notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -25,24 +25,32 @@ define([
          * @class KeyboardManager
          * @constructor
          * @param options {dict} Dictionary of keyword arguments :
-         *    @param options.events {$(Events)} instance 
+         *    @param options.events {$(Events)} instance
          *    @param options.pager: {Pager}  pager instance
          */
         this.mode = 'command';
+        this.view = 'tree';
```

Comment:
Actually, I'd rather rethink this to produce a cleaner architecture.

There should be a lower-level keyboard piece of code that handles the basic architecture of listening for shortcuts and responding to them, and then the notebook and the tree view should be able to build on that separately to listen for the shortcuts they want. You may have to split up and refactor some of the code to do this.

However, there's some big refactorings going on for 5.0, and I don't know if this code will still be used in its present form. @Carreau , @jdfreder : do either of you know what the fate of the keyboard interaction code will be?